### PR TITLE
Rails 5.1 support

### PR DESCRIPTION
**Pull request by mfazekas**
Rails 5.1 support + added ruby 2.4 and removed ruby 2.0 from travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,31 +5,33 @@ services: mongodb
...
   - 2.3.1
+  - 2.4.1
   - rbx-2
   - jruby
 gemfile:
   - gemfiles/active4.0.13.gemfile
   - gemfiles/active4.1.16.gemfile
   - gemfiles/active4.2.7.1.gemfile
   - gemfiles/active5.0.0.1.gemfile
+  - gemfiles/active5.1.gemfile
```
Comment (eitoball) [inline .travis.yml]: Please name this file to `active5.1.0.gemfile` as other files.

File: .travis.yml [hunk 2]
```diff
@@ -5,31 +5,33 @@ services: mongodb
...
   - gemfiles/rails4.0.13.gemfile
   - gemfiles/rails4.1.16.gemfile
   - gemfiles/rails4.2.7.1.gemfile
   - gemfiles/rails5.0.0.1.gemfile
+  - gemfiles/rails5.1.gemfile
```
Comment (eitoball) [inline .travis.yml]: Ditto

File: gemfiles/active5.1.gemfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "coveralls", :require => false
+gem "rubocop", :require => false
+gem "activesupport", "5.1"
```
Comment (eitoball) [inline gemfiles/active5.1.gemfile]: Please specify tiny version like '5.1.0'

File: gemfiles/rails5.1.gemfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "coveralls", :require => false
+gem "rubocop", :require => false
+gem "rails", "5.1"
```
Comment (eitoball) [inline gemfiles/rails5.1.gemfile]: Ditto

**Comment by mfazekas** [comment]
@eitoball thanks for the review, i'll fix the issues. On the failing build it's complaining about outdated gemfile.lock.

I assume some version of gem in lockfile was revoked from rubygems.org.

image

**Comment by mfazekas** [comment]
@eitoball pls review

**Comment by eitoball** [comment]
Thank you!

**Pull request merged**